packages/truffle-debugger/lib/controller/sagas/index.js

```diff
@@ -5,7 +5,8 @@ import { put, call, race, take, select } from 'redux-saga/effects';
 
 import { prefixName } from "lib/helpers";
 
-import * as trace from "lib/trace/sagas";
+import * as traceSagas from "lib/trace/sagas";
+import trace from "lib/trace/selectors";
```

Comment:
The convention is that sagas don't import selectors from an external app module. (This is why the controller selectors exist in the first place.)

Reply:
I see, didn't realize, will fix!

Reply:
(I'm guess you probably told me that at one point while explaining the architecture, but I'd entirely forgotten...)